src/main/java/com/conveyal/datatools/manager/persistence/TypedPersistence.java

```diff
@@ -142,6 +143,15 @@ public T getById (String id) {
         return mongoCollection.find().into(new ArrayList<>());
     }
 
+    /**
+     * Gets all objects with one of the IDs passed
+     * @param ids   List of IDs to return objects with
+     * @return      Objects in the collection with given IDs
+     */
+    public List<T> getByIds (List<String> ids) {
```

Comment:
Is this used anywhere?

Reply:
I think it might be once we switch to `Labels#feedSourceId`, but will leave this open as a reminder in case we don't use it

Reply:
This is now used to reduce the number of stream operations